src/status_im/contexts/wallet/send/events_test.cljs

```diff
@@ -163,3 +181,907 @@
+                                                            :tx-type transaction-type
+                                                            :collectible collectible
+                                                            :to-address to-address
+                                                            :receiver-networks receiver-networks
+                                                            :disabled-from-chain-ids
+                                                            disabled-from-chain-ids
+                                                            :from-locked-amounts from-locked-amounts
+                                                            :bridge-to-chain-id bridge-to-chain-id}}
+                                                :current-viewing-account-address address
+                                                :networks wallet-networks}
+                              :profile/profile {:test-networks-enabled? test-networks-enabled?}})
+        (is (match? expected-result (dispatch [event-id {:amount amount}])))))
+    (testing "when token is nil and transaction type is :tx/collectible-erc-721"
+      (let [transaction-type        :tx/collectible-erc-721
+            disabled-to-chain-ids   (filter (fn [chain-id]
+                                              (not (some #(= chain-id %)
+                                                         receiver-networks)))
+                                            network-chain-ids)
+            network-preferences     [(get-in collectible [:id :contract-id :chain-id])]
+            to-hex                  (fn [v] (send-utils/amount-in-hex v 0))
+            from-locked-amount      (update-vals from-locked-amounts to-hex)
+            sender-network-values   nil
+            receiver-network-values nil
+            request-params          [constants/send-type-erc-721-transfer
+                                     address
+                                     to-address
+                                     (to-hex amount)
+                                     (utils/format-token-id nil collectible)
+                                     to-token-id
+                                     disabled-from-chain-ids
+                                     disabled-to-chain-ids
+                                     network-preferences
+                                     constants/gas-rate-medium
+                                     from-locked-amount]
+            expected-result         {:db            {:wallet          {:ui {:send
+                                                                            {:amount amount
+                                                                             :loading-suggested-routes?
+                                                                             true
+                                                                             :sender-network-values
```

Comment:
Here the map is also difficult to read because it's too right-aligned, which forces zprint to add many undesirable line breaks (something a human wouldn't do I mean).